wasm/src/main/scala/ir2wasm/LibraryPatches.scala

```diff
@@ -32,10 +32,49 @@ object LibraryPatches {
     }
 
     derivedIRFiles.map { derived =>
-      derived.flatten ++ irFiles
+      derived.flatten ++ Seq(StackTraceIRFile) ++ irFiles
     }
   }
 
+  private val StackTraceIRFile: IRFile = {
+    val arrayOfSTERef = ArrayTypeRef(ClassRef("java.lang.StackTraceElement"), 1)
+
+    val classDef = ClassDef(
+      ClassIdent("java.lang.StackTrace$"),
+      NON,
+      ModuleClass,
+      None,
+      Some(ClassIdent(ObjectClass)),
+      Nil,
+      None,
+      None,
+      Nil,
+      List(
+        trivialCtor("java.lang.StackTrace$"),
+        MethodDef(
+          EMF, MethodIdent(m("captureJSError", List(ClassRef(ThrowableClass)), O)), NON,
+          List(paramDef("throwable", ClassType(ThrowableClass))), AnyType,
+          Some(
+            JSNew(JSGlobalRef("Error"), Nil)
+          )
+        )(EOH.withInline(true), NOV),
+        MethodDef(
+          EMF, MethodIdent(m("extract", List(O), arrayOfSTERef)), NON,
+          List(paramDef("jsError", AnyType)), ArrayType(arrayOfSTERef),
+          Some(
+            ArrayValue(arrayOfSTERef, Nil)
```

Comment:
https://github.com/scala-js/scala-js/blob/80eafa9ecb54f6763c0df32350c9c6cd8e7343c5/javalib/src/main/scala/java/lang/StackTrace.scala#L85-L94

This patched `extract` method always return `Nil`